Library/Formula/miller.rb

```diff
@@ -0,0 +1,25 @@
+class Miller < Formula
+  desc "Miller is like sed, awk, cut, join, and sort for name-indexed data such as CSV."
+  homepage "http://johnkerl.org/miller/doc/index.html"
+  url "https://github.com/johnkerl/miller/archive/v1.0.0.tar.gz"
+  sha256 "8e00d1e55609e2deb27537d385646ee4b3a4227070bdb1375ba7eff2a7e92708"
+
+  depends_on "flex"
+
+  def install
+    ENV.deparallelize
+    system "make"
+    # makefile hardcodes path to install the mlr executable as of 1.0.0.
+    # This is some creative wankery to get around that, in two steps:
+    # Step 1 - create the bin dir the makefile is expecting.
+    mkdir bin
+    # Step 2 - temporarily manipulate the value of $HOME to the cellar prefix homebrew provides.
+    system "HOME=\"#{prefix}\"; make install"
+  end
+
+  test do
+    # in lieu of any -v we could test a version from, lets just assert that there
```

Comment:
I copied the testing strategy here from the tmux recipe, and i'm hoping to add a '-v' to miller to tighten this up.  I figured homebrew was more concerned about making sure the binary is in place and responding to commands, not verifying functionality that ideally should be tested in the build for the binary itself.

Let me know if this adequately answers the question or if you'd still like a different testing strategy.